### DISCUSSION TITLE
Any plan for k>32 compressed plot?

### DISCUSSION THREAD

**Discussion by karlestira** [Chia-Network/chia-blockchain#16800] [Ideas]
chiapos has no compressed plot support. And bladebit has no k>32 plot support.

A larger k may save I/O on harvestor, and, maybe, can be used longer after k32 support stopped.

Slower chiapos is not a serious issues because I can replace existing plots one by one, the write lifespan can be solved with >1TB memory disk. But even if I have these hardware, I still cannot use k>32 compressed plot.

**Comment by BrandtH22** [comment]
Hey @karlestira , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

Yes there are plans to further develop bladebit to support higher than k32.
The chiapos plotter will likely not receive any additional updates and will not support compression.

It will likely be at least a couple years until the k32 size is no longer supported (still no plans at current to drop this support) so personally I would not be concerned about plotting k32 currently with the understanding that in a few years you may need to replot again.
